### PR TITLE
Refactor frontend code, remove Friendlycode

### PR DESCRIPTION
**Pull request by humphd**
This is only a first pass, we need to do more yet to rework things.  However, this dumps a lot of stuff we weren't using, but was still in the tree.

I need some help really testing this, esp. publish related stuff.

### CODE REVIEW

**Comment by humphd** [comment]
There's still some friendlycode stuff hanging around.  Anything you think I should dump in this PR?

```
.gitignore:public/friendlycode/css/editor.css
app.js:routes.friendlycodeRoutes(app);
public/stylesheets/editor.less:.friendlycode-base {
public/stylesheets/editor.less:.friendlycode-base img {
public/stylesheets/editor.less:.friendlycode-toolbar {
public/stylesheets/editor.less:.friendlycode-panes {
public/stylesheets/editor.less:.tipsy.friendlycode-base {
routes/index.js:    friendlycodeRoutes: function(app) {
routes/index.js:        res.render('friendlycode/templates/default-content.html', {
routes/index.js:        res.render('friendlycode/templates/error-dialog.html');
routes/index.js:        res.render('friendlycode/templates/confirm-dialog.html');
routes/index.js:        res.render('friendlycode/templates/publish-dialog.html');
routes/index.js:        res.render('friendlycode/templates/nav-options.html');
views/bramble.html:    <div class="friendlycode-toolbar">
views/bramble.html:        {% include 'friendlycode/templates/nav-options.html' %}
```

File: routes/index.js [hunk 1]
```diff
@@ -95,32 +93,9 @@ module.exports = function(utils, nunjucksEnv, appName) {
...
 
-      app.get( '/help-msg', function( req, res ) {
-        res.render('friendlycode/templates/help-msg.html');
-      });
-
-      app.get( '/error-msg', function( req, res ) {
-        res.render('friendlycode/templates/error-msg.html');
-      });
-
       app.get( '/nav-options', function( req, res ) {
```
Comment (sedge) [inline routes/index.js]: This is the only file we still use from the `friendlycode/templates` folder, and it doesn't need it's own middleware handler. If you stuff the file in `/views` and change the reference where it's rendered in `bramble.html` that should be enough to make the entire `friendlycodeRoutes` function unnecessary.

Comment (humphd) [inline routes/index.js]: Awesome, this is exactly what I wanted to know.  I'll do just that!

**Comment by sedge** [comment]
Nuke:
```
.gitignore:public/friendlycode/css/editor.css
app.js:routes.friendlycodeRoutes(app);
public/stylesheets/editor.less:.friendlycode-panes {
public/stylesheets/editor.less:.tipsy.friendlycode-base {
```

See my other comment regarding the `friendlycodeRoutes` function.

The css selectors `.friendlycode-base`, `.friendlycode-base img` and `.friendlycode-toolbar` are still being used, but should probably be renamed wherever you see them. Long live friendlycode.

**Pull request merged**